crates/holochain_sqlite/src/db.rs

```diff
@@ -445,6 +445,10 @@ pub struct DbKindCache(pub Arc<DnaHash>);
 /// Specifies the environment used by a Conductor
 pub struct DbKindConductor;
 
+#[derive(Clone, Debug, PartialEq, Eq, Hash, derive_more::Display)]
+/// Specifies the environment used to witness nonces.
+pub struct DbKindNonce;
+
```

Comment:
- it probably won't be large because we delete all expired nonces every time we witness a new nonce, so the database can't get larger than however many nonces can be seen in the expiry window
- if we want to move the data to a different db i suggest doing it as a followup as the migration is trivial due to all nonce data being transient, but 
- that could potentially result in worse performance (idk?) if the other db has to set aside a db column for what will be mostly null values due to not having recently seen a nonce for the majority of known agents, so i think that's a premature optimisation and maybe even a regression unless/until we have real data